package.json

```diff
@@ -16,21 +16,23 @@
 	"license": "Apache-2.0",
 	"author": "Prismic <contact@prismic.io> (https://prismic.io)",
 	"sideEffects": false,
-	"type": "module",
```

Comment:
I'm curious why we removed `"type": "module"` here. Is it specific to this SDK of should be removed on others too?